public/js/poll/view.js

```diff
@@ -63,7 +63,7 @@
 		},
 		insertPoll: function(poll, callback) {
 			View.parsePoll(poll, function(html) {
-				$('#post-container .post-row[data-index="0"] .post-content').prepend(html);
+				$('.clearfix[data-index="0"] .content').append(html);
```

Comment:
This is where I'm worried about it only supporting the Persona theme. Putting an anchor on all ```.clearfix``` tags seems shady but I couldn't find those id's anywhere (```#post-container``` & ```.post-row```)